packages/pulumi/src/executors/preview/preview.impl.ts

```diff
@@ -24,7 +25,7 @@ export default async function creatExecutor(
       options.stack && `--stack=${options.stack}`
     ]),
     {
-      cwd: sourceRoot,
+      cwd: options.cwd ?? sourceRoot,
```

Comment:
My prefference would go to aligning this more with how the other packages do it if you don't mind, see [here](https://github.com/TriPSs/nx-extend/blob/master/packages/strapi/src/executors/build/build.impl.ts#L47)

Reply:
Interesting - this seems to go against `nx`'s way of doing it. They are using absolute paths, and also calling the working directory option `cwd`.

Knowing this, do you still want to go that way?

Alternatively I can change all your options too to be compliant with NX's standard.

Reply:
Changing the option in the other packages would cause breaking changes in those so i'm not really in favour of doing that 😅

Reply:
There we go! I think I got all of them. Thanks for the feedback!